tests/regression/lp-1898038/task.yaml

```diff
@@ -0,0 +1,23 @@
+summary: docker-support and multipass-support should mount base /etc/apparmor.d
+prepare: |
+    tests.cleanup prepare
+    snap pack test-docker-support-app
+    tests.cleanup defer rm -f test-docker-support-app_1_all.snap
+    snap install --dangerous ./test-docker-support-app_1_all.snap
+    tests.cleanup defer snap remove --purge test-docker-support-app
+    snap pack test-multipass-support-app
+    tests.cleanup defer rm -f test-multipass-support-app_1_all.snap
+    snap install --dangerous ./test-multipass-support-app_1_all.snap
+    tests.cleanup defer snap remove --purge test-multipass-support-app
+execute: |
+    # wait a bit to ensure the mount namespace exists
+    sleep 5
+    # check that within the snap mount namespace /etc/apparmor is mounted
+    # and comes from a snap
+    test "$(nsenter -m/run/snapd/ns/test-docker-support-app.mnt mount | grep -c '/etc/apparmor\(.d\)\? type squashfs')" -eq 2
+    test "$(nsenter -m/run/snapd/ns/test-multipass-support-app.mnt mount | grep -c '/etc/apparmor\(.d\)\? type squashfs')" -eq 2
+    # also check the snaps loaded their apparmor profiles
+    test "$(aa-status | grep -c 'snap.test-\(docker\|multipass\)-support.app.test-\(docker\|multipass\)-support')" -eq 2
+restore: |
+    tests.cleanup restore
```

Comment:
It would be great to unload the test profile here.

Reply:
Thanks - I just added this in abf0f62